lib/anyway/settings.rb

```diff
@@ -107,5 +110,8 @@ def matching_env?(env)
 
     # Tracing is enabled by default
     self.tracing_enabled = true
+
+    # By default, use ANYWAY_SUPPRESS_VALIDATIONS
+    self.suppress_required_validations = ENV["ANYWAY_SUPPRESS_VALIDATIONS"]
```

Comment:
Let's be more specific here:

```suggestion
    self.suppress_required_validations = %w[1 t true y yes].include?(ENV["ANYWAY_SUPPRESS_VALIDATIONS"])
```